Dockerfile_tesseract_training

```diff
@@ -36,20 +38,45 @@ RUN cd .. && \
     cd ..  && \
     git clone https://github.com/tesseract-ocr/tesstrain.git && \
     cd tesstrain && \
-    python3 -m venv ocr && \
     mkdir data && cd data  && \
     cd /usr/local/share/tessdata && wget https://github.com/tesseract-ocr/tessdata_best/raw/main/eng.traineddata && \
     wget https://github.com/tesseract-ocr/tessdata_fast/blob/65727574dfcd264acbb0c3e07860e4e9e9b22185/osd.traineddata && \
     mv /usr/local/share/tessdata/ /train
 
-# Setting the TESSDATA_PREFIX prefix
-ENV TESSDATA_PREFIX=/train/tessdata
+RUN mkdir -p /train/langdata && cd /train/langdata && \
+    wget https://raw.githubusercontent.com/tesseract-ocr/langdata_lstm/main/radical-stroke.txt && \
+    wget https://raw.githubusercontent.com/tesseract-ocr/langdata_lstm/main/common.punc  && \
+    wget https://raw.githubusercontent.com/tesseract-ocr/langdata_lstm/main/font_properties && \
+    wget https://raw.githubusercontent.com/tesseract-ocr/langdata_lstm/main/Latin.unicharset  && \
+    wget https://raw.githubusercontent.com/tesseract-ocr/langdata_lstm/main/Latin.xheights  && \
+    mkdir eng && cd eng && \
+    wget https://raw.githubusercontent.com/tesseract-ocr/langdata/main/eng/eng.training_text && \
+    wget https://raw.githubusercontent.com/tesseract-ocr/langdata/main/eng/eng.punc && \
+    wget https://raw.githubusercontent.com/tesseract-ocr/langdata/main/eng/eng.numbers && \
+    wget https://raw.githubusercontent.com/tesseract-ocr/langdata/main/eng/eng.wordlist
+
+RUN mkdir -p /usr/local/share/fonts/journal && \
+    wget https://dl.dafont.com/dl/?f=journal -O /tmp/journal.zip && \
+    unzip /tmp/journal.zip -d /usr/local/share/fonts/journal
+
+# Setting the TESSDATA_PREFIX
```

Comment:
TESSDATA_PREFIX never actually got set like it did in the old container so this actually breaks the workflow if a user tries to use the container

Reply:
I updated both the docker-compose.yml file and the Dockerfile with the requested changes, thanks for the gist that was super helpful :)